### PR TITLE
Fix: Node run commands

### PR DESCRIPTION
**Pull request by EugenWay**
https://github.com/gear-tech/wiki/pull/104

### CODE REVIEW

File: docs/node/setting-up.mdx [hunk 1]
```diff
@@ -147,16 +153,16 @@ It doesn't matter if you downloaded the prebuild binary or built it yourself. Be
...
 
 To run Gear node in dev mode use the following command:
 
 ```bash
-gear-node --dev --tmp
+./gear-node --dev --tmp
 ```
 
-## Comands Flags and Options
+## Comands Flags and Options§
```
Comment (shamilsan) [inline docs/node/setting-up.mdx:165]:
```suggestion
## Comands Flags and Options
```

File: docs/node/setting-up.mdx [hunk 2]
```diff
@@ -15,6 +15,12 @@ This guide covers the steps required to install and run the Gear node.
 
 There are two ways to get started with the Gear node. First, you can download a pre-built binary file and run it, or second, you can compile the binary file and configure it by yourself. Using a ready-made build is a quick and convenient way to get started as you can skip the installation of Rust and all its dependencies and you can also skip the node compiling process. On the other hand, if you decide that you want to go through the process of creating your development node, it may take about twenty minutes or more depending on your hardware.
 
+:::info
+### System requirements
+
+Gear node doesn't have any special hardware requirements exepting SSD
```
Comment (shamilsan) [inline docs/node/setting-up.mdx:21]:
```suggestion
Gear node doesn't have any special hardware requirements excepting SSD
```

**Review by shamilsan — APPROVED** [review]
Looks good except for some minor typos.

File: docs/node/setting-up.mdx (hunk 2 above)
Comment (EugenWay) [reply docs/node/setting-up.mdx]: fixed

File: docs/node/setting-up.mdx (hunk 1 above)
Comment (EugenWay) [reply docs/node/setting-up.mdx]: fixed

**Pull request merged**